cql/src/main/java/io/stargate/cql/impl/CqlImpl.java

```diff
@@ -91,14 +95,18 @@ public void start() {
       }
     }
 
-    ClientMetrics.instance.init(servers, metrics.getRegistry("cql"));
+    int metricUpdateRate =
+        Integer.parseInt(System.getProperty("stargate.cql.metrics.updateRate", "10000"));
```

Comment:
nit: rename `updateRate` to `updatePeriodSeconds` for the same of clarity? The "rate" for updating once every 10 sec would be 0.1 (unit 1/s), right?

Reply:
Done.